src/tracing/index.ts

```diff
@@ -6,6 +6,7 @@ import { initializeAnalytics, OriginApplication } from '@uniswap/analytics'
 import { SharedEventName } from '@uniswap/analytics-events'
 import { isSentryEnabled } from 'utils/env'
 import { getEnvName, isProductionEnv } from 'utils/env'
+import { v4 as v4uuid } from 'uuid'
```

Comment:
For consistency with the rest of the codebase:

```suggestion
import { v4 as uuidv4 } from 'uuid'
```